### DISCUSSION TITLE
Is there a way to have a hosts file so I can ssh to remote servers by name instead of IP?

### DISCUSSION THREAD

**Discussion by pmi123** [termux/termux-app#3700] [Q&A] [answered]
Is there a way to have a hosts file so I can ssh to remote servers by name instead of IP? My phone is a Pixel 8 Pro and is not rooted, nor do I want to root it. Thanks!

**Comment by agnostic-apollo** [comment, marked as answer]
https://linuxize.com/post/using-the-ssh-config-file/

**Reply by pmi123** [reply]
Thanks!
